04-onion-routing.md

```diff
@@ -238,6 +240,10 @@ The requirements ensure consistency in responding to an unexpected
 `outgoing_cltv_value`, whether it is the final node or not, to avoid
 leaking its position in the route.
 
+`attributable_error` is a marker record that specifies that the failure message
```

Comment:
Should we add directions for the reader to handle the case where the sender has set `attributable_error` but the proceeding node doesn't actually support `option_attributable_error`?

ie for: A - B - C 
* Sets `attributable_error` (either by mistake or maliciously) 
* B doesn't support attributable errors
* C will try to fail back with a new format error, which B doesn't understand

Reply:
I don't think it is necessary. The only party that is affected by this is A itself. Node B will just encrypt the failure data without any interpretation. It makes no sense, but B doesn't care that A cannot read the message.